UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -45,6 +45,7 @@ public GameObject ControllingObject
 		get
 		{
 			if (IsGhosting) return this.gameObject;
+			if (IDActivelyControlling is NetId.Invalid or NetId.Empty) return this.gameObject;
```

Comment:
does `Invalid` also cover cases where it hasn't been initialized yet? or is it `Empty` by default on start?

Reply:
yeah, I think those are only two invalid states for network IDs